### ISSUE TITLE
Support JDK 11 and above

### ISSUE THREAD

**Issue by solcyr** [cossme/grinder#44]
Grinder 4.X is limited on Java 8 due to the fact Grinder extends the Application ClassLoader by merging the URLs.
ince Java9, ApplicationClassLoader is not an URL ClassaLoader anymore.

blockingClassLoader should be reimplemented and lot of dependencies must be upgraded

**Comment by jonadem** [comment]
Is this bug generating this kind of error ?

> 2020-11-05 15:50:17,290 ERROR agent: Fatal Error - incompatible version of Java (Java(TM) SE Runtime Environment 11.0.8+10-LTS: Java HotSpot(TM) 64-Bit Server VM (11.0.8+10-LTS, mixed mode) on Windows 10 amd64 10.0)%nThe Grinder 4.0.0 requires at least Java SE 6.

**Comment by solcyr** [comment]
Yes. This is the message. Did you check with the branch 5 ?

**Comment by jonadem** [comment]
No, and I won't push tests further. I found another solution for what I was looking for.
